server/schedulers/hot_region.go

```diff
@@ -87,8 +89,9 @@ type balanceHotRegionsScheduler struct {
 	types       []BalanceType
 
 	// store id -> hot regions statistics as the role of leader
-	stats *storeStatistics
-	r     *rand.Rand
+	stats      *storeStatistics
+	r          *rand.Rand
+	scoreInfos *ScoreInfos
```

Comment:
How about adding some comments to explain how we use it?

Reply:
how about `ScoreInfos stores storeID and score of all stores.`